examples/inputs/turbine_definitions/iea_10MW.yaml

```diff
@@ -0,0 +1,164 @@
+turbine_type: 'iea_10MW'
```

Comment:
Is this used? I think we are mapping this definition to the name in the main input file, so could possibly be removed.

Reply:
It is, in that it makes it easier to do some of the farm expansion functions, but wasn't completely necessary with the previous code. Now that the turbine definition has been collapsed in the input file, I do believe it is necessary.